freeipa.spec.in

```diff
@@ -248,6 +248,7 @@ Requires: %{name}-server-common = %{version}-%{release}
 Requires: %{name}-common = %{version}-%{release}
 Requires: python2-ipaclient = %{version}-%{release}
 Requires: python-ldap >= 2.4.15
+Requires: python-lxml
```

Comment:
Exactly, I removed the dependency on lxml from ipalib. Only ipaserver depends on lxml.